src/main/resources/application.conf

```diff
@@ -1,4 +1,5 @@
 proxy = "https://cas-legacy.subscriptions.guardianapis.com/"
+proxyNew = "http://subs-cas-CASLegac-ZWI0H7Z9JY8T-1440512801.eu-west-1.elb.amazonaws.com/public"
```

Comment:
No but you weren't here to discuss, basically I see no point in the CAS application explicitly routing to /public only to have jetty rewrites back to route. I was going to suggest changing the CAS routes instead of having the confusion of additional jetty config, what are you thoughts here?